documentation/basics/headless.md

```diff
@@ -89,75 +88,75 @@ the command (as shown in the example above).
+- `"Pixel Probabilities"`, which exports the pixel prediction images of the pixel classification part of that workflow. Only valid if you specified an `.ilp` `--project` that was created with the  "Pixel Classification + Object Classification" workflow.
 
-Per default, ilastik exports a label image of the object class predictions.
-This is equivalent to setting `--export_source="Object Predictions"` in the command line.
-Use the `--export_source` flag to specify the output of the processing explicitly.
-If the flag is set to `--export_source="Object Probabilities"`, ilastik will export a multi-channel image volume of object prediction probabilities
-instead of a label image (one channel for each prediction class).
-Finally, if you happen to be using the "Pixel Classification + Object Classification" workflow which combines pixel classification
-and object classification into a single workflow, you may export the pixel prediction images by providing the
-`--export_source="Pixel Probabilities"` flag.
 
-If you provide path for the `--table_filename` output, ilastik will export a `.csv` file of the computed object features that were used during classification, indexed by object id.
+[Object Classification Workflow]: {{site.baseurl}}/documentation/objects/objects.html
+
+Depending on which variant of the [Object Classification Workflow] you used to create your `.ilp` project file, you may need to provide more than one input image for each volume of data you want to process (e.g. "Raw Data" and "Segmentation Image" for , or "Raw Data" and "Prediction Maps"). These input images correspond to the tabs in the `Input Data` applet of the Object Classification Workflow and must be provided on the command-line.  To specify which is which, prefix the list of input files with either `--raw_data`, `--segmentation_image`, or `--prediction_maps` accordingly. Here's an example of invoking ilastik with an Object Classification Workflow that takes a Segmentation Image as input:
+
+    $ ./run_ilastik.sh --headless \
+                       --project=MyObjClassFromPredictMap.ilp \
+                       --table_filename=/tmp/exported_object_features.csv \
+                       --export_source="Object Predictions" \
+                       --raw_data "my_grayscale_stack_1/*.png" \
+                       --segmentation_image my_unclassified_objects_1.h5/binary_segmentation_volume
```

Comment:
adding an example with, or at least mentioning `--prediction_maps` would be nice